portfolio_prices.py

```diff
@@ -0,0 +1,65 @@
+import investpy as inv
+from datetime import datetime
+import pandas as pd
+import numpy as np
+
+today = datetime.today().strftime("%d/%m/%Y")
+
+
+def get_assets_data_frames(assets, asset_function, country, start_date, end_date):

```

Comment:
`python
def get_assets(...) -> pd.DataFrame:
`
I think typing the functions is better than writting the name of the function with the type

Reply:
same thing for the following functions